jib-core/src/main/java/com/google/cloud/tools/jib/configuration/Port.java

```diff
@@ -83,9 +112,15 @@ public boolean equals(Object other) {
 
   @Override
   public int hashCode() {
-    return Objects.hash(port, protocol);
+    return Objects.hash(port, protocol.toString());
```

Comment:
Why the `toString` here? Just curious.

Reply:
Since `protocol` is just an enum to better organize string constants, I thought it'd make more sense to hash the actual string rather than the enum constants.

Reply:
It shouldn't affect the result either way so it's probably fine, however I feel like adding the `toString` makes it marginally less readable.

Reply:
I think we can just make protocol a string.